### PR TITLE
PROCESSINGS: Retrieve User By Credentials

### LINKED ISSUE #154: PROCESSINGS: Retrieve User By Credentials
RetrieveUserByCredentials(string username, string password)

### PR DESCRIPTION
**Pull request by Javlonbek-dev**
Closes #154

### CODE REVIEW

File: Tarteeb.Api.Tests.Unit/Services/Processings/Users/UserProcessingsServiceTests.Logic.cs [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+using System.Linq;
+using Tarteeb.Api.Models;
+using Xunit;
+
+namespace Tarteeb.Api.Tests.Unit.Services.Processings.Users
+{
+    public partial class UserProcessingsServiceTests
+    {
+        [Fact]
+        public void ShouldAddCountryIfNotExistAsync()
```
Comment (sarvarbek-dev) [inline Tarteeb.Api.Tests.Unit/Services/Processings/Users/UserProcessingsServiceTests.Logic.cs:18]: why is it written addCountry

Comment (Javlonbek-dev) [reply Tarteeb.Api.Tests.Unit/Services/Processings/Users/UserProcessingsServiceTests.Logic.cs]: done

File: Tarteeb.Api.Tests.Unit/Services/Processings/Users/UserProcessingsServiceTests.Logic.cs [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+namespace Tarteeb.Api.Tests.Unit.Services.Processings.Users
+{
+    public partial class UserProcessingsServiceTests
+    {
+        [Fact]
+        public void ShouldAddUserIfNotExistAsync()
+        {
+            // given
+            string inputEmail = GetrandomString();
+            string inputpassword = GetrandomString();
```
Comment (sarvarbek-dev) [inline Tarteeb.Api.Tests.Unit/Services/Processings/Users/UserProcessingsServiceTests.Logic.cs:22]: use camel case

File: Tarteeb.Api/Services/Processings/Users/UserProcessingService.cs [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+        private readonly IUserService userService;
+
+        public UserProcessingService(IUserService userService) =>
+            this.userService = userService;
+
+        public User RetrieveUserByCredentails(string email, string password)
+        {
+            IQueryable<User> allUser = this.userService.RetrieveAllUsers();
+
+            return allUser
```
Comment (ElbekDeveloper) [inline Tarteeb.Api/Services/Processings/Users/UserProcessingService.cs:19]: Bu metodda hali exception handling va validation yo'q

**Review by ElbekDeveloper — CHANGES_REQUESTED** [review]

**Pull request closed without merging**